### ISSUE TITLE
Direct Messages donot load properly.

### ISSUE THREAD

**Issue by mr-turdoide** [baedert/corebird#770]
The direct messages that I deleted years ago is re appearing only in Corebird but not in twitter.com
Is here anyone with same issue?

**Comment by Vistaus** [comment]
Yes, I reported that once but there was no solution available: https://github.com/baedert/corebird/issues/641

**Comment by baedert** [comment]
The other problem here is that the twitter API simply returns deleted messages, which it doesn't show on twitter.com. In my testing that only happened when I deleted a whole conversation from twitter.com. To actually get the DMs to be deleted, I had to delete them individually.

**Comment by mr-turdoide** [comment]
@baedert There is no option to delete the DMs in corebird, isn't it?

**Comment by baedert** [comment]
Nope

**Comment by mr-turdoide** [comment]
How am I supposed to do that then? I couldnot find any ways to do so.

**Comment by baedert** [comment]
Well it just doesn't exist in Corebird right now. You can get manually rid of DMs, but you still have to delete each message on its own since the twitter api doesn't have a way of deleting conversations or bulk-deleting DMs in any way.
